File: app/src/main/java/se/gustavkarlsson/skylight/android/background/providers/AuroraFactorsProvider.java
package se.gustavkarlsson.skylight.android.background.providers;

import android.location.Location;

import org.threeten.bp.Duration;

import se.gustavkarlsson.skylight.android.models.AuroraFactors;

public interface AuroraFactorsProvider {
	AuroraFactors getAuroraFactors(Location location, Duration timeout);
}

File: app/src/main/java/se/gustavkarlsson/skylight/android/background/providers/AuroraFactorsProvider.kt
package se.gustavkarlsson.skylight.android.background.providers

import android.location.Location

import org.threeten.bp.Duration

import se.gustavkarlsson.skylight.android.models.AuroraFactors

internal interface AuroraFactorsProvider {
    fun getAuroraFactors(location: Location, timeout: Duration): AuroraFactors
}
